### PR TITLE
Updated Installation Steps

### PR DESCRIPTION
**Pull request by susanmdouglas-aws**
Updated compiling-babelfish-from-source.md with installation steps from Emanuel (based on jdbc_unit_tests.yml).

Signed-off-by: susanmdouglas <susandou@amazon.com>

### Issues Resolved
This commit updates the installation, but contains a number of FIXMEs and needs to be tested before committing.

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _installation/compiling-babelfish-from-source.md [hunk 1]
```diff
@@ -6,261 +6,211 @@ nav_order: 2
...
 ## Downloading the source code
 
-Before building Babelfish, you will need to download the source code.  Babelfish source code is separated into two repositories that contain:
+Before building Babelfish, you will need to download the source code.  Babelfish source code is stored in the following repositories: FIXME-are_these_still_the_same_repo_names_?
 
 - the [PostgreSQL database engine](https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish) source code, with changes that provide the protocols, language parsers, and features required by Babelfish. 
 - the [extensions](https://github.com/babelfish-for-postgresql/babelfish_extensions) that support the T-SQL protocol, the T-SQL language, the TDS Protocol, and so on.
 
+FIXME - Do we need to ask the user to navigate into a specific directory location before invoking these commands so the build commands later in this page work?
+
```
Comment (nasbyj) [inline _installation/compiling-babelfish-from-source.md:20]: No

File: _installation/compiling-babelfish-from-source.md [hunk 2]
```diff
@@ -6,261 +6,211 @@ nav_order: 2
...
-- [Flex 2.6.4](https://github.com/westes/flex)
-- [Libxml2](http://xmlsoft.org/) development libraries
-- [Open SSL](https://www.openssl.org/) development libraries
-- [Readline](https://tiswww.cwru.edu/php/chet/readline/rltop.html) development libraries
-- [Zlib](https://zlib.net/)
-- [OSSP uuid](http://www.ossp.org/pkg/lib/uuid/) development libraries
-- [pkg-config](https://linux.die.net/man/1/pkg-config)
-- [ICU](https://icu.unicode.org/) development libraries
-- [Bison 3.0.5 or higher](https://www.gnu.org/software/bison/)
+FIXME - Do we need to create a user named 'runner'? If so:
```
Comment (nasbyj) [inline _installation/compiling-babelfish-from-source.md:32]: No, we should remove all those references and simply warn the user that Postgres can not be run as root.

File: _installation/compiling-babelfish-from-source.md [hunk 3]
```diff
@@ -6,261 +6,211 @@ nav_order: 2
...
 # Compiling Babelfish from source
 
-This section will walk you through the steps required to create a working Babelfish installation on Linux. 
+This section will walk you through the steps required to create a working Babelfish installation on an Ubuntu 20.04 Linux host. Please note that the steps may vary on other operating systems, but the overall process is roughly the same. 
 
 
 ## Downloading the source code
 
-Before building Babelfish, you will need to download the source code.  Babelfish source code is separated into two repositories that contain:
+Before building Babelfish, you will need to download the source code.  Babelfish source code is stored in the following repositories: FIXME-are_these_still_the_same_repo_names_?
```
Comment (nasbyj) [inline _installation/compiling-babelfish-from-source.md:14]: We should start using the release tarballs.

**Review by nasbyj — CHANGES_REQUESTED** [review]

File: _installation/compiling-babelfish-from-source.md (hunk 2 above)
Comment (susanmdouglas-aws) [reply _installation/compiling-babelfish-from-source.md:32]: Those references are gone... I mention creating a non-root user in 30/34, but don't say explicitly why it must be non-root.

File: _installation/compiling-babelfish-from-source.md (hunk 3 above)
Comment (3manuek) [reply _installation/compiling-babelfish-from-source.md:14]: What about including another file with "Compiling and Installing from tarballs"? Steps slightly vary, but it'll be more readable for users.  IMHO both methods should coexist.  cc/ @nasbyj

**Comment by 3manuek** [comment]
@susanmdouglas-aws I've made my review and done the modifications, could you please review back?

**Review by 3manuek — APPROVED** [review]

**Review by gopinathpai — DISMISSED** [review]

**Review by nasbyj — APPROVED** [review]

**Review by nasbyj — APPROVED** [review]

**Review by 3manuek — APPROVED** [review]

**Pull request merged**